nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ListenHTTP.java

```diff
@@ -159,11 +160,20 @@ public AllowableValue getAllowableValue() {
             .build();
     public static final PropertyDescriptor AUTHORIZED_DN_PATTERN = new PropertyDescriptor.Builder()
         .name("Authorized DN Pattern")
-        .description("A Regular Expression to apply against the Distinguished Name of incoming connections. If the Pattern does not match the DN, the connection will be refused.")
+        .description("A Regular Expression to apply against the Subject's Distinguished Name of incoming connections. If the Pattern does not match the Subject DN, " +
+                "the the processor will respond with a status of HTTP 403 Forbidden.")
         .required(true)
-        .defaultValue(".*")
+        .defaultValue(MATCH_ALL)
         .addValidator(StandardValidators.REGULAR_EXPRESSION_VALIDATOR)
         .build();
+    public static final PropertyDescriptor AUTHORIZED_ISSUER_DN_PATTERN = new PropertyDescriptor.Builder()
+            .name("Authorized Issuer DN Pattern")
```

Comment:
It is preferred to use separate `name` / `displayName` attributes for new properties, like in case of "Multipart Request Max Size":
```
        .name("multipart-request-max-size")
        .displayName("Multipart Request Max Size")
```
Could you please use the same pattern here?